### ISSUE TITLE
Need ability to set VC2019 redistributable

### ISSUE THREAD

**Issue by quasar-kyle-2** [itchio/itch#2360]
... in the .itch.toml file.  Should be a self explanitory issue, but more info here https://itch.io/t/611136/how-do-i-add-visual-c-2019-redistributable

**Comment by quasar-kyle-2** [comment]
I was told this wasn't clear enough.   Here is another explanation.

hello. any reason why the VC 2019 redistributable isnt a supported prereq?  https://itch.io/docs/itch/integrating/prereqs/vc.html
i've had numerous issues reported where my game crashes on startup and I have to tell people to go download the runtime (https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads  ) , because I cant specify it as a prereq

TLDR, GAME CRASHES ON STARTUP DUE TO THIS

**Comment by fasterthanlime** [comment]
Hi there!

The issue was clear enough, I just needed to get to it :)

Both `vcredist-2019-x86` and `vcredist-2019-x64` have been added, automatic prereq detection should also be enabled for those (when a game links against `msvcp160.dll`).

**Comment by fasterthanlime** [comment]
Hi there!

The issue was clear enough, I just needed to get to it :)

Both `vcredist-2019-x86` and `vcredist-2019-x64` have been added, automatic prereq detection should also be enabled for those (when a game links against `msvcp160.dll`).

**Comment by fasterthanlime** [comment]
Hi there!

The issue was clear enough, I just needed to get to it :)

Both `vcredist-2019-x86` and `vcredist-2019-x64` have been added, automatic prereq detection should also be enabled for those (when a game links against `msvcp160.dll`).
